content/04.results.md

```diff
@@ -25,6 +25,10 @@ Figure {@fig:mutational-landscape-overview}B-C show concordance of these samples
 ### Recurrence and co-occurrence of mutations
 
 This section will discuss the genes and regions that are repeatedly mutated within and between cancer types.
-The occurrence of mutations in affecting particular genes, separated by cancer,  type is shown in Figure {@fig:occurrence-plots}A, with significant co-occurrence across all types and within types with sufficient sample sizes illustrated in Figure {@fig:occurrence-plots}B.
+The occurrence of mutations in affecting particular genes, separated by tumor type is shown in Figure {@fig:occurrence-plots}A, with significant co-occurrence across all types illustrated in Figure {@fig:occurrence-plots}B.
 
-![Occurrence and co-occurrence of mutations](images/figure_sketches/occurrence-fig-sketch.jpg){#fig:occurrence-plots}
+![Occurrence and co-occurrence of nonsynonymous mutations for the 50 most commonly mutated genes across all tumor types.
+A) Counts of nonsynonymous mutations, colored by tumor type.
+B) Cooccurence and mutual exclusivity of nonsynonymous mutations between genes.
+The cooccurrence score is defined as $I(-\log_{10}(P))$ where $P$ is defined by Fisher's exact test and $I$ is 1 when mutations cooccur more often than expected and -1 when exclusivity is more common.
+](https://github.com/AlexsLemonade/OpenPBTA-analysis/raw/master/figures/pngs/mutation_cooccurrence_figure.png){#fig:occurrence-plots}
```

Comment:
Looking at the examples here: https://github.com/greenelab/iscb-diversity-manuscript/blob/6d7595f564405156d0d3a633847a2f7cc03b57f7/content/20.results.md

You may want:
```suggestion

](https://github.com/AlexsLemonade/OpenPBTA-analysis/raw/master/figures/pngs/mutation_cooccurrence_figure.png){#fig:occurrence-plots}
```
So that the figure legend shows up?

Reply:
Looking at the examples on that page, it seems like my version has the figure in the github render, but not the legend, while the one suggested allows the figure *legend* to show up when github renders, but not the figure itself. Either seems to work in the manubot-rendered documents: https://greenelab.github.io/iscb-diversity-manuscript/

Reply:
Okay works for me